packages/analytics-nextjs/src/context.tsx

```diff
@@ -42,6 +43,28 @@ export function useAnalyticsContext() {
     return analyticsContext;
 }
 
+function PlausibleWrapperMaybe({ newsroom, children }: PropsWithChildren<Pick<Props, 'newsroom'>>) {
```

Comment:
No idea how to test if `PlausibleProvider` is properly rendered based on the Newsroom settings, since it doesn't render any markup to test against. 
I could add an element that would only render in testing environment, but that feels like a bad solution 😕 
Curious what you think @kudlajz

Reply:
What if you add a `TestingComponent` that uses `usePlausible` hook? I would assume that the hook should not work if `PlausibleProvider` is not present but I couldn't find an answer from their Github page.

Reply:
Nope, doesn't work. The function is always defined, even when the provider is not present. Since the hook doesn't expose anything else, I can't test with it.

Tried querying for the inserted scripts, but that didn't succeed too.

Reply:
Ended up using a dirty solution, but it ended up being the only one that worked.